README.md

```diff
@@ -22,9 +22,27 @@ is not fully covered by the testsuite. If you encounter any mistakes or
 omissions please let me know by posting an issue, or even better: send me a
 pull request with a fix.
 
+## History
+
+This library was originally developed by authors listed below. However, the 
+github account with this project was deleted. 
+Now it has been reestablished by developers of [Amethyst engine](https://github.com/amethyst/)
```

Comment:
I think it's prematurely to say that. Library can exist and be usable on its own, and so far only you made changes after its resurrection.